### PR TITLE
docs: outside contributions

### PR DESCRIPTION
**Pull request by moog16**
fixes https://github.com/material-components/material-components-web-react/issues/440

### CODE REVIEW

File: docs/contributions/README.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+Pull requests should meet the following criteria:
+
+* **PRs should be focused!** PRs should focus on fixing one issue or an additional feature. Anything extra requires another PR.
+* If there is an existing [issue](https://github.com/material-components/material-components-web-react/issues) please refer to it in the description of your PR.
+* Please also add notes/description about what your changes are trying to achieve (or anything you've learned). Brevity appreciated.
+
+### Process
+
+Before opening a PR, it should be up to date with targeted release (rc<release_number> ie. rc7.0, rc8.0, etc.). In most cases it will never be branched from _master_.
```
Comment (bonniezhou) [inline docs/contributions/README.md]: We should add `0.` in front of the versions in the branch name moving forward, like `rc0.7.0`, `rc0.8.0`.
So it's not confusing if we ever release `1.0.0` lol 😱

File: docs/contributions/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+* If the changes you want to perform are under 10 - 20 lines of code, please open a [pull request](https://github.com/material-components/material-components-web-react/pulls).
+* Anything larger or change to an API will require an issue to be opened first.
+
+### Commit Message Format
+
+The final commit message to the mdc-foo package, for GitHub issue 1234, should look like this:
+
+```
+# for a fix to an issue
+fix(foo): Short description of fix
```
Comment (bonniezhou) [inline docs/contributions/README.md]: Replace `foo` with `component-name`. Here and 2 below

File: docs/contributions/README.md (hunk 1 above)
Comment (moog16) [reply docs/contributions/README.md]: ya true - just thought it was easier to type lol

File: docs/contributions/running-tests.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+Tests are an important part of coding. Please refer to the following instructions when contributing to the repository.
+
+**Ensure you're on Node v10**
+If you need help updating, please see [NVM](https://github.com/creationix/nvm#installation). :heart: NVM.
+
+## Screenshot Tests
+
+Please refer to [screenshot tests](../screnshot-tests.md).
+
+## Unit Tests
```
Comment (bonniezhou) [inline docs/contributions/running-tests.md]: Add a section for running lint. Or include a note at the top that tells people to just run `npm test` for all tests

**Pull request merged**